lib/composer.js

```diff
@@ -462,7 +462,7 @@ class Composer {
       info
     })
 
-    const result = await this.runQueries(queries, order)
+    const result = await this.runQueries(queries, order, excludeKeys(context.reply.request.headers, ['content-type', 'content-length']))
```

Comment:
Is there a better way of getting a hold of the request headers? (one may have thought `context.request.headers` would have worked)